draft-ietf-quic-qpack.md

```diff
@@ -1126,6 +1126,13 @@ HTTP_QPACK_DECODER_STREAM_ERROR (0x202):
 
 TBD.
 
+While the negotiated limit on the dynamic table size accounts for much of the
+memory that can be consumed by a QPACK implementation, data which cannot be
+immediately sent due to flow control is not affected by this limit.
+Implementations MUST limit the size of unsent data, especially on the decoder
```

Comment:
Does a MUST really make sense here?  I mean, it seems like the implementation's problem if they have an unbounded amount of memory buffered.

Reply:
I think that having a MUST for the decoder stream might make sense (as the peer controls what is to be sent on the decoder stream). OTOH I share the view that having MUST limit + MUST close if exceeded is a bit off for the encoder stream.